script/start-odc.sh

```diff
@@ -110,15 +110,16 @@ function init_jvm_options() {
     fi
     local log_options="-Dlog4j.configurationFile=${app_log_config_file} -Dodc.log.directory=${app_log_directory}"
     local work_dir_options="-Duser.dir=${ODC_WORK_DIR:-${current_work_directory}}"
-    app_options="${log_options} ${work_dir_options}"
+    local plugin_options="-Dplugin.dir=${plugin_directory}"
```

Comment:
plugins initialzied before spring cloud config, so we should use app options (`-Dblabla` ) instead of app args(`--blabla` )